test/tests/val-2a/val-2a_base.i

```diff
@@ -0,0 +1,187 @@
+high_dt_max = 100
```

Comment:
If you are only modeling one case, you don't need to combine two input file, let's discuss potentially merging them back together.